jsonnet/dashboards/apps/openstack.libsonnet

```diff
@@ -0,0 +1,1672 @@
+          options: {
+            '0': { text: 'Down', color: $._config.grafanaDashboards.color.red },
+            '1': { text: 'Up', color: $._config.grafanaDashboards.color.green },
+            '-1': { text: '-' },
+          },
+        })
+
+        .addTarget(
+          prometheus.target('openstack_object_store_up{job=~"$job",instance=~"$instance", cluster="$cluster"} OR on() vector(-1)')
+        );
+
+      local containerInfraStatus =
+        statPanel.new(
+          title='Container infrastructure Status',
+          datasource='$datasource',
+          unit='string',
+          colorMode='background',
+          graphMode='none',
+          reducerFunction='lastNotNull',
+        )
+
+        .addThresholds([
+          { color: 'transparent', value: null },
+        ])
+
+        .addMapping({
+          type: 'value',
+          options: {
+            '0': { text: 'Down', color: $._config.grafanaDashboards.color.red },
+            '1': { text: 'Up', color: $._config.grafanaDashboards.color.green },
+            '-1': { text: '-' },
+          },
+        })
+
+        .addTarget(
+          prometheus.target(
+            'openstack_container_infra_up{job=~"$job",instance=~"$instance", cluster="$cluster"} OR on() vector(-1)'
+          )
+        );
+
+      local heatStatus =
+        statPanel.new(
+          title='Heat Status',
+          datasource='$datasource',
+          unit='string',
+          colorMode='background',
+          graphMode='none',
+          reducerFunction='lastNotNull',
+        )
+
+        .addThresholds([
+          { color: 'transparent', value: null },
+        ])
+
+        .addMapping({
+          type: 'value',
+          options: {
+            '0': { text: 'Down', color: $._config.grafanaDashboards.color.red },
+            '1': { text: 'Up', color: $._config.grafanaDashboards.color.green },
+            '-1': { text: '-' },
+          },
+        })
+
+        .addTarget(
+          prometheus.target(
+            'openstack_heat_up{job=~"$job",instance=~"$instance", cluster="$cluster"} OR on() vector(-1)')
+        );
+
+      local gnocchiStatus =
+        statPanel.new(
+          title='Gnocchi Status',
+          datasource='$datasource',
+          unit='string',
+          colorMode='background',
+          graphMode='none',
+          reducerFunction='lastNotNull',
+        )
+
+        .addThresholds([
+          { color: 'transparent', value: null },
+        ])
+
+        .addMapping({
+          type: 'value',
+          options: {
+            '0': { text: 'Down', color: $._config.grafanaDashboards.color.red },
+            '1': { text: 'Up', color: $._config.grafanaDashboards.color.green },
+            '-1': { text: '-' },
+          },
+        })
+
+        .addTarget(
+          prometheus.target(
+            'openstack_gnocchi_up{job=~"$job",instance=~"$instance", cluster="$cluster"} OR on() vector(-1)')
+        );
+
+      local troveStatus =
```

Comment:
Please remove this status panel cause we are not using it, and also remove `heat` and `container infra`